llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -389,6 +389,105 @@ InstructionCost RISCVTTIImpl::getShuffleCost(TTI::ShuffleKind Kind,
   // give a more accurate cost than falling back to generic scalable codegen.
   // TODO: Each of these cases hints at a modeling gap around scalable vectors.
   if (isa<FixedVectorType>(Tp)) {
+    MVT LegalVT = LT.second;
```

Comment:
Is there a test for the RISC-V cost changes?

Reply:
Added